### PR TITLE
AWS-AccessAnalyzer GA

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2121

## Description
Moved AWS-AccessAnalyzer from Beta to GA

## Does it break backward compatibility?
   - [x] Yes
       - some context path was changed from the beta version

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]
Impressive work

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 1]
```diff
@@ -28,16 +28,20 @@ configuration:
...
   name: roleArn
   required: false
   type: 0
 - display: Role Session Name
+  additionalinfo: A descriptive name for the assumed role session. For example, xsiam-IAM.integration-Role_SESSION
   name: roleSessionName
   required: false
   type: 0
 - display: Role Session Duration
+  additionalinfo: 'The maximum length of each session in seconds. The integration will have the permissions assigned only when the session is initiated and for the defined duration.'
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:41]:
```suggestion
  additionalinfo: The maximum length of each session in seconds. The integration will have the permissions assigned only when the session is initiated and for the defined duration.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 2]
```diff
@@ -62,6 +66,26 @@ configuration:
   required: false
   type: 4
   hidden: true
+- display: First fetch time interval
+  additionalinfo: The time range to consider for the initial data fetch. (<number>
+    <unit>, e.g., 2 minutes, 2 hours, 2 days, 2 months, 2 years). Default is 3 days.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:71]:
```suggestion
    <unit>, e.g., 2 minutes, 2 hours, 2 days, 2 months, 2 years). 
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 3]
```diff
@@ -62,6 +66,26 @@ configuration:
...
+  defaultvalue: 3 days
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+- display: Maximum number of incidents to fetch every time
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:77]:
```suggestion
- display: Maximum number of incidents to fetch every time.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 4]
```diff
@@ -62,6 +66,26 @@ configuration:
...
+  defaultvalue: '25'
+  hidden: false
+  name: max_fetch
+  required: false
+  type: 0
+- display: Fetch Analyzer ARN
+  name: analyzerArn
+  additionalinfo: The ARN to fetch findings for
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:85]:
```suggestion
  additionalinfo: The ARN to fetch findings for.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 5]
```diff
@@ -70,14 +94,25 @@ configuration:
   name: proxy
   required: false
   type: 8
-- display: Fetch Analyzer ARN
-  name: analyzerArn
-  additionalinfo: The ARN to fetch findings for
-  defaultvalue: ""
+- display: Timeout
+  name: timeout
+  additionalinfo: The time in seconds till a timeout exception is reached. You can
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:99]:
```suggestion
  additionalinfo: The time in seconds until a timeout exception is reached. You can
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 6]
```diff
@@ -70,14 +94,25 @@ configuration:
...
+    specify just the read timeout (for example 60) or also the connect timeout followed
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:100]:
```suggestion
    specify just the read timeout (for example 60) or also the connect timeout following
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 7]
```diff
@@ -70,14 +94,25 @@ configuration:
...
+    after a comma (for example 60,10). If a connect timeout is not specified, a default
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:101]:
```suggestion
    a comma (for example 60,10). If a connect timeout is not specified, a default
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 8]
```diff
@@ -70,14 +94,25 @@ configuration:
...
+    of 10 second will be used.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:102]:
```suggestion
    of 10 seconds will be used.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 9]
```diff
@@ -111,9 +146,12 @@ script:
     execution: false
     name: aws-access-analyzer-list-analyzers
     outputs:
-    - contextPath: AWS.AccessAnalyzer.Analyzers.arn
+    - contextPath: AWS.AccessAnalyzer.Analyzers
       description: List of Analyzers and their details.
       type: Unknown
+    - contextPath: AWS.AccessAnalyzer.Analyzers.arn
+      description: The Analyzer arn.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:153]:
```suggestion
      description: The Analyzer ARN.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 10]
```diff
@@ -170,9 +208,15 @@ script:
     execution: false
     name: aws-access-analyzer-list-analyzed-resource
     outputs:
-    - contextPath: AWS.AccessAnalyzer.Analyzers.Resource
+    - contextPath: AWS.AccessAnalyzer.Resource
       description: List of analyzed resources.
       type: Unknown
+    - contextPath: AWS.AccessAnalyzer.Resource.resourceArn
+      description: The resource Arn.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:215]:
```suggestion
      description: The resource ARN.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 11]
```diff
@@ -170,9 +208,15 @@ script:
...
+      type: String
+    - contextPath: AWS.AccessAnalyzer.Resource.analyzerArn
+      description: The analyzer Arn.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:218]:
```suggestion
      description: The analyzer ARN.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 12]
```diff
@@ -240,9 +284,15 @@ script:
...
     outputs:
-    - contextPath: AWS.AccessAnalyzer.Findings
+    - contextPath: AWS.AccessAnalyzer.Finding
       description: Access Analyzer findings.
       type: Unknown
+    - contextPath: AWS.AccessAnalyzer.Finding.id
+      description: The Finding ID.
+      type: String
+    - contextPath: AWS.AccessAnalyzer.Finding.analyzerArn
+      description: The analyzer arn.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:294]:
```suggestion
      description: The analyzer ARN.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 13]
```diff
@@ -240,9 +284,15 @@ script:
     execution: false
     name: aws-access-analyzer-list-findings
     outputs:
-    - contextPath: AWS.AccessAnalyzer.Findings
+    - contextPath: AWS.AccessAnalyzer.Finding
       description: Access Analyzer findings.
       type: Unknown
+    - contextPath: AWS.AccessAnalyzer.Finding.id
+      description: The Finding ID.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:291]:
```suggestion
      description: The finding ID.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 14]
```diff
@@ -285,9 +335,15 @@ script:
     execution: false
     name: aws-access-analyzer-get-analyzed-resource
     outputs:
-    - contextPath: AWS.AccessAnalyzer.Analyzers.Resource
+    - contextPath: AWS.AccessAnalyzer.Resource
       description: Analyzed resource detail.
       type: Unknown
+    - contextPath: AWS.AccessAnalyzer.Resource.id
+      description: The Resource id.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:342]:
```suggestion
      description: The resource ID.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 15]
```diff
@@ -285,9 +335,15 @@ script:
...
+      type: String
+    - contextPath: AWS.AccessAnalyzer.Resource.analyzerArn
+      description: The analyzer arn.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:345]:
```suggestion
      description: The analyzer ARN.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 16]
```diff
@@ -330,9 +386,15 @@ script:
     execution: false
     name: aws-access-analyzer-get-finding
     outputs:
-    - contextPath: AWS.AccessAnalyzer.Analyzers.Finding
+    - contextPath: AWS.AccessAnalyzer.Finding
       description: Finding details.
       type: Unknown
+    - contextPath: AWS.AccessAnalyzer.Finding.id
+      description: The Finding ID.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:393]:
```suggestion
      description: The finding ID.
```

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 17]
```diff
@@ -330,9 +386,15 @@ script:
...
+      type: String
+    - contextPath: AWS.AccessAnalyzer.Finding.analyzerArn
+      description: The analyzer arn.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:396]:
```suggestion
      description: The analyzer ARN.
```

**Comment by ShirleyDenkberg** [comment]
@ilappe Please review my comments in the yml and then regenerate the readme.
@DeanArbel Doc review completed.

**Pull request merged**